spec/custom_matchers.rb

```diff
@@ -0,0 +1,12 @@
+RSpec::Matchers.define :belong_to do |expected|
+  match { |actual| (actual.to_a.uniq - expected.to_a.uniq).empty? }
+end
+
+RSpec::Matchers.define :contain_from do |expected|
+  match do |actual|
+    actual = actual.to_a.map { |x| x.data.to_a }.flatten.uniq
+    expected.map!(&:values) unless expected.first.is_a? Array
+    expected = expected.flatten.uniq
+    (actual - expected).empty?
+  end
+end
```

Comment:
I kinda understand this, but what is line 8, for example?.. And I am not sure that name is communicative enough: in fact, it suits for checking dataframes (right?), so its name probably should communicate that?..

And it looks even weaker than previous one, checking "ok, some data kinda should be somewhere", without even checking the data would be in right columns or whatnot?..